project_euler/problem_14/sol2.py

```diff
@@ -47,8 +47,8 @@ def collatz_sequence(n):
 def solution(n):
     """Returns the number under n that generates the longest Collatz sequence.
 
-    >>> solution(1000000)
-    {'counter': 525, 'largest_number': 837799}
+    #>>> solution(1000000)
+    #{'counter': 525, 'largest_number': 837799}
```

Comment:
`#<space>` please on both of these lines
Also, please add a comment on why these lines have been commented out.

Reply:
Done.